ci/check-style.sh

```diff
@@ -24,7 +24,10 @@ fi
 # This script assumes it is running the top-level google-cloud-cpp directory.
 
 find google/cloud -name '*.h' -print0 \
-  | xargs -0 awk 'BEGINFILE {
+  | xargs -0 awk 'BEGIN {
```

Comment:
This awk script is long enough that it might be worth either A) indenting it one level further than the code above it, or B) adding line comments around it, e.g. `# BEGIN awk` and `# END awk`